src/openms/source/METADATA/ProteinIdentification.cpp

```diff
@@ -338,69 +340,55 @@ namespace OpenMS
 
   Size ProteinIdentification::computeCoverage(const std::vector<PeptideIdentification>& pep_ids)
```

Comment:
Can we make `computeCoverage()` `void` instead of returning `0`?

When to use an exit status and when an exception?
http://nedbatchelder.com/text/exceptions-vs-status.html